static/_redirects

```diff
@@ -430,10 +430,220 @@
+
+# More ref/app paths
+/ref/app /guides/app/ 301
+/ref/app/features/panels/bar-plot /guides/app/features/panels/bar-plot/ 301
+/ref/app/features/panels/parallel-coordinates /guides/app/features/panels/parallel-coordinates/ 301
+/ref/app/features/panels/parameter-importance /guides/app/features/panels/parameter-importance/ 301
+/ref/app/features/panels/weave /guides/app/features/panels/weave/ 301
+/ref/app/features/panels/weave/embedding-projector /guides/app/features/panels/weave/embedding-projector/ 301
+/ref/app/features/custom-charts /guides/app/features/custom-charts/ 301
+/ref/app/features/tags /guides/app/features/tags/ 301
+/ref/app/pages/workspaces /guides/track/workspaces/ 301
+
+# API paths
+/ref/api /ref/python/public-api/api/ 301
+/api /ref/python/public-api/api/ 301
+/artifacts/api /ref/python/sdk/classes/artifact/ 301
+
+# Python ref paths
+/ref/python/run /ref/python/sdk/classes/run/ 301
+/ref/python/save /ref/python/public-api/run#save/ 301
+/ref/python/watch /ref/python/sdk/functions/watch/ 301
+/ref/python/login /ref/python/sdk/functions/login/ 301
+/ref/python/artifact /ref/python/sdk/classes/artifact/ 301
+/ref/python/agent /ref/python/sdk/functions/agent/ 301
+/ref/python/sweep /ref/python/sdk/functions/sweep/ 301
+/ref/python/finish /ref/python/sdk/functions/finish/ 301
+/ref/release-notes/releases/index.xml/ /ref/release-notes/index.xml/ 301
+/ref/release-notes/releases/ /ref/release-notes/ 301
+/ref/release-notes/0.56.0/ /ref/release-notes/releases/archived/0.56.0/ 301
+/ref/release-notes/0.57.2/ /ref/release-notes/releases/archived/0.57.2/ 301
+
+## More language-specific paths (static, must come before dynamic rules)
+# Japanese specific redirects
+/ja/guides/app/features/teams /ja/guides/app/features/teams/ 301
+/ja/guides/app/pages/workspaces /ja/guides/track/workspaces/ 301
+
+/ja/guides/app/pages/workspaces/ /ja/guides/track/workspaces/ 301
+
+# Korean specific redirects  
+/ko/guides/track/launch /ko/launch/ 301
+/v/ko/ref/run/config /ko/ref/python/run/ 301
+
+# Edge cases
+/guides/data-vis/tables)です）。 /guides/models/tables/ 301
+/guides/artifacts/meow /guides/artifacts/ 301
+
+# More specific support paths
+/support/permissions_agent_require_kubernetes /support/agent-require-kubernetes/ 301
+/support/permissions_agent_require_kubernetes/ /support/agent-require-kubernetes/ 301
+/support/secrets_jobsautomations_instance_api_key_wish_directly_visible /support/secrets-api-key/ 301
+/support/index_launch/ /support/launch/ 301
+
+# App path
+/app /guides/app/ 301
+
+# Guides app pages that don't exist - redirect to main app section
+/guides/app/pages/run-page/ /guides/app/ 301
+/guides/app/pages/run-page /guides/app/ 301
+/guides/app/pages/project-page/ /guides/app/ 301
+/guides/app/settings-page/ /guides/app/ 301
+/guides/app/settings-page /guides/app/ 301
+/guides/app/settings-page/user-settings /guides/app/ 301
+/guides/app/settings-page/team-settings /guides/app/ 301
```

Comment:
Setting pages exist..they were moved within the "models" folder when it was consolidated by major products:

e.g. https://docs.wandb.ai/guides/models/app/settings-page/